zng/cc.rs

```diff
@@ -137,13 +133,25 @@ pub fn build_zlib_ng(target: &str, compat: bool) {
             "inftrees",
             "insert_string",
             "insert_string_roll",
-            "slide_hash",
             "trees",
             "uncompr",
             "zutil",
         ],
     );
 
+    cfg.append(
+        Some("arch/generic"),
```

Comment:
Can you fill me in a little why this was necessary? Did these files move?

Reply:
Yes, these files moves from `/` to `/arch/generic`, see this change https://github.com/zlib-ng/zlib-ng/compare/2.1.x...2.2.x#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a